trillian/ctfe/requestlog.go

```diff
@@ -64,6 +64,12 @@ type RequestLog interface {
 	// LeafHash will be called once for get proof by hash requests with the
 	// requested hash value (if the parameters parse correctly).
 	LeafHash(context.Context, []byte)
+	// SCT will be called once when the server is about to issue an SCT to a
+	// client. This need not be called if the submission process fails before an
```

Comment:
/need not/should not/?
i.e. "only set this when the associated leaf is known to have been successfully queued and the intention is to issue the SCT"?